src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Distinct.txt

```diff
@@ -0,0 +1,5 @@
+>> CountRows(Distinct([{a: 1}, {a: 1}, {a: 3}], Value.a))
```

Comment:
.txt file is great. 
We'll eventually need to add more cases here,  including corner cases like blank/error.

Reply:
Added a bunch of cases. :)